server/szurubooru/api/info_api.py

```diff
@@ -20,7 +20,10 @@ def _get_disk_usage() -> int:
     for dir_path, _, file_names in os.walk(config.config['data_dir']):
```

Comment:
I'm guessing this is the root cause of the problem:

A race condition occurs between when `os.walk(config.config['data_dir'])` is called and when all the file sizes are summed up at the end of the loop. If someone deletes a post during this period, then that will cause the missing file exception.

Not an issue with this PR specifically, but I'm wondering if there's a smarter way to sidestep the problem entirely...